examples/github-complete/main.tf

```diff
@@ -72,10 +72,9 @@ module "atlantis" {
   trusted_principals = ["ssm.amazonaws.com"]
 
   # Atlantis
-  atlantis_github_user        = var.github_user
-  atlantis_github_user_token  = var.github_token
-  atlantis_repo_allowlist     = ["github.com/${var.github_owner}/*"]
-  atlantis_allowed_repo_names = var.allowed_repo_names
```

Comment:
this variable is also used in the webhook sub-module - we will need to somehow accommodate that as well with this change otherwise it breaks the webhooks. even though atlantis doesn't use this param anymore, we still need to repo names for the webhooks

```
 # module.github_repository_webhook.github_repository_webhook.this[0] must be replaced
-/+ resource "github_repository_webhook" "this" {
      + etag       = (known after apply)
      ~ id         = "xxxxxxxxx" -> (known after apply)
      ~ repository = "scritchity-scratch" -> "github.com/clowdhaus/*" # forces replacement
      ~ url        = "https://api.github.com/repos/clowdhaus/scritchity-scratch/hooks/xxxxxxxxx" -> (known after apply)
        # (2 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }
```

Reply:
@bryantbiggs thanks for your comment, but I think you are missing, that instead of `atlantis_allowed_repo_names` is removed, it is renamed to `atlantis_repo_allowlist` for consistency naming in submodules as well.

Please check this commit https://github.com/terraform-aws-modules/terraform-aws-atlantis/pull/221/commits/38496544a3b96ccf71659ddf97f297654a391113

Now both module and submodules are using the same naming and also the same list of repositories.

Is there any reason, that those two repository lists (white-list for atlantis + webhooked repositories) should be separated?